sdk/eventhub/Azure.Messaging.EventHubs/src/Properties/AssemblyInfo.cs

```diff
@@ -4,6 +4,11 @@
 using System.Diagnostics.CodeAnalysis;
 using System.Runtime.CompilerServices;
 
+#pragma warning disable AZC0011 // Internal visible to product libraries effectively become public API and have to be versioned appropriately
```

Comment:
This comment should list specific APIs that the friend assembly will call. Also, please file an issue to develop static analysis tools to allow granting friendship on fine granularity level, i.e. if friend assembly calls any API other than explicitly allowed the tools will flag it as an error.

Reply:
I'm not sure that I understand either part of the ask.  Are you asking that I just note what `internal` members are being called from the other package in the comment?   

Is this issue that you're asking for on our SDK tools or are you thinking something for the .NET team?    I think the context of the issue is that you want to be able to detect (via comment?) the "blessed" internal members that will be called and warn if anything else is called?